rzup/install

```diff
@@ -15,218 +16,227 @@
 
 set -euo pipefail
```

Comment:
script will fail if:
- any command fails (-e)
- any variable is undefined (-u) ()
- any part of a pipeline fails (-o pipefail)